client/api/src/main/java/io/smallrye/graphql/client/typesafe/api/TypesafeGraphQLClientBuilder.java

```diff
@@ -81,6 +82,8 @@ default TypesafeGraphQLClientBuilder headers(Map<String, String> headers) {
 
     TypesafeGraphQLClientBuilder subprotocols(WebsocketSubprotocol... subprotocols);
 
+    TypesafeGraphQLClientBuilder clientModel(ClientModels.ClientModel clientModel);
```

Comment:
Only add this to our implementation of the builder (`VertxTypesafeGraphQLClientBuilder`) please, in the future we want to propose the general client API to the MP specification, and this method surely won't be part of the proposal. 

(Yeah, so to use it, it will be necessary to directly use `VertxTypesafeGraphQLClientBuilder`, or cast to it)